cmd/influxd/server_integration_test.go

```diff
@@ -433,9 +476,9 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"logs","columns":["time","event"]}]}]}`,
 		},
 		{
-			name:     "missing measurement with `GROUP BY *`",
-			query:    `select load from "%DB%"."%RP%".missing group by *`,
-			expected: `{"results":[{"error":"measurement not found: \"%DB%\".\"%RP%\".\"missing\""}]}`,
+			name:          "missing measurement with `GROUP BY *`",
+			query:         `select load from "%DB%"."%RP%".missing group by *`,
+			expectPattern: `.*measurement not found: .*missing.*`,
```

Comment:
Ah! I though the line of code was you telling me to check the source. :-)

That said, I don't think the errors being returned to users should include source code lines. It's too much noise, and the information is of no use to them. It's too low-level, and I have never seen this in practise, short of stack traces. Or am I missing something?